src/libraries/Microsoft.Extensions.DependencyInjection/tests/DI.External.Tests/Microsoft.Extensions.DependencyInjection.ExternalContainers.Tests.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetFrameworkCurrent)</TargetFrameworks>
+    <EnableDefaultItems>true</EnableDefaultItems>
+    <SignAssembly>false</SignAssembly>
```

Comment:
Why can't we sign this assembly? We use a Test key.

Reply:
These tests are using 3ed parties packages which including non-signed assemblies.

Reply:
Added NoWarn so we could sign this as a test assembly